src/components/Book.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+
+class Book extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+    this.value = 0.64;
+  }
+
+  render() {
+    const { title, author } = this.props;
+    return (
+      <div className="container my-3 px-5 border border-opacity-10 border-dark rounded-1 bg-light d-flex justify-content-between align-items-center flex-fill">
+        <div style={{ width: '30%' }}>
+          <div className="py-3">
+            <span className="text-muted">Action</span>
+            <h3 className="fw-bold">{title}</h3>
+            <span className="" style={{ color: '#5aa8ee' }}>{author}</span>
+          </div>
+          <div>
+            <ul className="list-unstyled">
+              <li className="list-inline-item" style={{ color: '#5aa8ee' }}>Comments</li>
+              <li className="list-inline-item text-muted">|</li>
+              <li className="list-inline-item" style={{ color: '#5aa8ee' }}>Remove</li>
+              <li className="list-inline-item text-muted">|</li>
+              <li className="list-inline-item" style={{ color: '#5aa8ee' }}>Edit</li>
+            </ul>
+          </div>
+        </div>
+        <div style={{ width: 180, height: 180 }} className="align-items-center d-flex gap-2">
```

Comment:
- [ ] Let's try to avoid inline styling as it is not best practice. Kindly move all styles to an external CSS file.